cms/static/coffee/spec/models/upload_spec.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+define(["js/models/uploads"], FileUpload =>
+
+    describe("FileUpload", function() {
+        beforeEach(function() {
+            return this.model = new FileUpload();
+        });
+
+        it("is unfinished by default", function() {
+            return expect(this.model.get("finished")).toBeFalsy();
+        });
+
+        it("is not uploading by default", function() {
+            return expect(this.model.get("uploading")).toBeFalsy();
+        });
+
+        it("is valid by default", function() {
+            return expect(this.model.isValid()).toBeTruthy();
+        });
+
+        it("is valid for text files by default", function() {
+            const file = {"type": "text/plain", "name": "filename.txt"};
```

Comment:
This is not using our ESLint checker either, as the keywords should not be quoted. I wonder if ESLint is ignoring these directories because they used to contain CoffeeScript? You might find that they start to have errors again when you move them to a different directory.